src/uct/base/uct_md.c

```diff
@@ -458,9 +460,15 @@ static ucs_status_t uct_md_attr_v2_init(uct_md_h md, uct_md_attr_v2_t *md_attr)
         return status;
     }
 
-    /* Component name + data */
+    memcpy(global_id, md_attr->global_id,
+           sizeof(md_attr->global_id) - component_name_length);
+    memcpy(md_attr->global_id, md->component->name, component_name_length);
+    memcpy(UCS_PTR_BYTE_OFFSET(md_attr->global_id,
+                               component_name_length),
+           global_id, sizeof(md_attr->global_id) - component_name_length);
```

Comment:
IMO it's simpler to just do it in UCT IB.
Besides, it doesn't always have to be the component name in the prefix. We just decided it as convention. But it could be anything unique per component. So no need to have it in the "base" code.

Reply:
done